content/news/XXXX-XX-XX-mixxx-2-5-beta-released.md

```diff
@@ -0,0 +1,51 @@
+title: Mixxx 2.5 beta released
+authors: Evelynne Veys
+tags: 2.5, beta, release announcement
+comments: yes
+status: draft
+
+#### Dear Mixxx-ers
+
+Funny things happen in spring, birds start nesting and lay eggs, and so did the Mixxx developers.
+A bit late for Easter but it is a magnificent giant gift filled with smaller gifts: Mixxx 2.5 beta!
+You can find the details about the new version below.
+
+#### Appeal
+
+Before releasing 2.5 as the next stable release, we need it to be tested thoroughly, something we can't do on our own.
+After some time looking at the same release and searching for solutions, we might forget minor bugs or problems.  
+That's why we need **YOU** to help us. The more people testing this Beta and reporting eventual bugs to the [bug tracker](https://github.com/mixxxdj/mixxx/issues), the faster the beta can be promoted to stable.  
+In case you're scared of messing up your neatly configured PC and losing your holy data is your Haloween nightmare, read the new wiki article [Safeguard Your Mixxx Data](https://github.com/mixxxdj/mixxx/wiki/Safeguard-your-Digital-DJ-Data).
+It explains how to creata a timestamped image, how to back up your data and offers tips and tricks.  
+A beta release contains the solutions for problems that occurred and were notified by users who tested the alpha release.
+The beta release is a general check before the new version can be released.  
+The Mixxx database and settings of Mixxx 2.4.x stable are equal to those in Mixxx 2.5 beta, which means you can install the 2.5 over the 2.4 and vice versa without a problem.  
+After you have successfully tested this 2.5 beta for yourself, you can continue using this version in your bedroom or even during live DJ-ing, and thus benefit from the new features.  
```

Comment:
I don't think we need to explain what a beta is.

```suggestion
The Mixxx database and settings format of Mixxx 2.4.x stable have not changed in Mixxx 2.5 beta, which means you can install the 2.5 over the 2.4 and revert back without a problem.  
We don't recommend using the beta version in public performances, but you can if you like!  
```

Reply:
> I don't think we need to explain what a beta is.

Maybe not for regular visitors / github users / people who experiment with software, but for a lot of people alpha, beta, release, ... sounds as extraterrestrial rocket science. 
Perhaps a wiki article about the Mixxx development process and used terms could help?